docs/source/intro/troubleshooting.rst

```diff
@@ -269,6 +269,13 @@ To test if a the computer does not produce spurious output, run (after configuri
 
 which checks and, in case of problems, suggests how to solve the problem.
 
+.. note::
+
+    If, for whatever reason, the methods explained above do not work, you can also opt to change the computer configuration to not use a login shell.
+    During `verdi computer configure`, set `-no-use-login-shell` or when asked to use a login shell, set it to `False`.
+    This will tell AiiDA to not use a login shell when connecting, which may potentially prevent the spurious output from being printed.
+    Note, however, that this may also prevent other code from being executed, such as certain environment variables being set.
```

Comment:
```suggestion
    If the methods explained above do not work, you can configure AiiDA to not use a login shell when connecting to your computer, which may prevent the spurious output from being printed:
    During ``verdi computer configure``, set ``-no-use-login-shell`` or when asked to use a login shell, set it to ``False``.
    Note, however, that this may result in a slightly different environment, since `certain startup files are only sourced for login shells <https://unix.stackexchange.com/a/46856/155909>`_.
```